web/src/pm_list.js

```diff
@@ -33,7 +36,6 @@ export function close() {
     private_messages_collapsed = true;
     $("#toggle_private_messages_section_icon").removeClass("fa-caret-down");
     $("#toggle_private_messages_section_icon").addClass("fa-caret-right");
-
```

Comment:
Don't remove blank lines in code not touched by your PR. It's distracting for reviewers.